tests/datasets/test_events.py

```diff
@@ -35,9 +35,16 @@ def test_column_expr(self):
         )
 
         # All tag keys expression
-        assert column_expr(
-            self.dataset, "tags_key", deepcopy(query), ParsingContext()
-        ) == ("(arrayJoin(tags.key) AS tags_key)")
+        q = Query({"granularity": 86400, "selected_columns": ["tags_key"]}, source,)
+        assert column_expr(self.dataset, "tags_key", q, ParsingContext()) == (
+            "(arrayJoin(tags.key) AS tags_key)"
+        )
+
+        # Selected tags
+        q = Query(
+            {"granularity": 86400, "selected_columns": ["tags_key[tags1, tags2]"]},
+            source,
+        )
```

Comment:
This assignment is unused in any test (and to my understanding should no longer be supported in this change?)

Reply:
This comes from an earlier version where we were parsing tags_key[....]
Just forgot to remove.